bricks/presentational-bricks/src/general-carousel/GeneralCarousel.tsx

```diff
@@ -108,6 +115,21 @@ export function GeneralCarousel({
       prevArrow={<LeftOutlined />}
       nextArrow={<RightOutlined />}
       dotPosition={dotPosition}
+      onInit={() =>
+        useLazyLoad &&
+        setEleDisplay(
+          `.slick-slide:not([data-index="0"]):not([data-index="${data.length}"])`,
+          "none"
+        )
+      }
+      beforeChange={(currentSlide, nextSlide) =>
+        useLazyLoad &&
+        setEleDisplay(`.slick-slide[data-index="${nextSlide}"]`, "block")
+      }
+      afterChange={(currentSlide) =>
+        useLazyLoad &&
+        setEleDisplay(`.slick-slide[data-index="${--currentSlide}"]`, "none")
```

Comment:
似乎可以不用在前面的 `onInit` 来给首尾节点单独做处理？因为这里可以根据 current slide 来得到准确的 previous slide

```suggestion
        setEleDisplay(`.slick-slide[data-index="${currentSlide === 0 ? data.length - 1 : currentSlide - 1}"]`, "none")
```